test/test_tc2halide.cc

```diff
@@ -24,23 +24,14 @@
 #include "tc/core/polyhedral/schedule_isl_conversion.h"
 #include "tc/core/polyhedral/scop.h"
 #include "tc/core/tc2halide.h"
-#include "tc/core/utils/dlpack.h"
+#include "tc/core/tensor.h"
 
 using namespace tc;
 using namespace std;
-using namespace tc::dlutils;
 
 struct TC2Isl : public ::testing::Test {
   void SetUp() {}
-  void Check(const string& tc, const std::vector<long>& inputSizes) {
-    auto ctx = getCPUDLContext();
-    DLDataType dtype;
-    dtype.code = kDLFloat;
-    dtype.bits = 32;
-    dtype.lanes = 1;
-    auto UPtr = makeDLTensorWithSizes(ctx, dtype, inputSizes);
-    std::vector<const DLTensor*> inputs{UPtr.get()};
-
```

Comment:
Please mention why you removed this.
Bonus points if you can do it in a separate commit.